packages/material-ui-lab/src/Breadcrumbs/Breadcrumbs.d.ts

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { StandardProps } from '@material-ui/core';
+
+export interface BreadcrumbsProps
+  extends StandardProps<React.HTMLAttributes<HTMLDivElement>, BreadcrumbsClassKey> {
```

Comment:
```suggestion
  extends StandardProps<React.HTMLAttributes<HTMLElement>, BreadcrumbsClassKey> {
```
`nav` is used by default which is only a `HTMLElement`.